factories_ZA/sampleList.py

```diff
@@ -48,6 +50,12 @@
     'ggZH_HToBB_ZToNuNu_M125_13TeV_powheg_pythia8_Summer16MiniAODv2'
 ]
 
+#WGamma
+samples_dict["WGamma"] = [
+    'WGToLNuG_TuneCUETP8M1_13TeV-madgraphMLM-pythia8_Summer16MiniAODv2',
+    'WGToLNuG_TuneCUETP8M1_13TeV-amcatnloFXFX-pythia8_extended_ext0_plus_ext1_plus_ext2_plus_ext3'
```

Comment:
`ext0` is actually the sample without the extension, so it means it is duplicated here!